op-guide/docker-across-multiple-hosts.md

```diff
@@ -0,0 +1,166 @@
+# Run TiDB in Docker across Multiple Hosts
+
+This page shows you a example of manually deploying a TiDB cluster using docker on multiple machines.
```

Comment:
+This page shows you an example of how to manually deploy a TiDB cluster using Docker on multiple machines.